.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Report a bug report to help us improve the module.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+<!-- **IMPORTANT!**
+Before reporting a bug, please make sure that you have read through our documentation and you think your problem is indeed an issue related to our module. -->
+
+### Version
+
+@nuxt/content: <!-- ex: v2.0.0 -->
+nuxt: <!-- ex: v3.0.0-rc.3 -->
+
+### Reproduction Link
+<!--
+A minimal test case based on one of:
+- a fork of https://codesandbox.io/s/github/nuxt/content/tree/main/examples/essentials/hello-world
```

Comment:
Shall we add StackBlitz as well?

Reply:
It should only be stackblitz.

Will update the PR.